#!/usr/bin/env python
PACKAGE = "vision"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("rgb", int_t, 0, "RGB", 0, 0, 3)
gen.add("h_min", int_t, 0, "Hue Low", 0, 0, 255)
gen.add("s_min", int_t, 0, "Saturation Low", 0, 0, 255)
gen.add("v_min", int_t, 0, "Value Low", 0, 0, 255)
gen.add("h_max", int_t, 0, "Hue Max", 0, 0, 255)
gen.add("s_max", int_t, 0, "Saturation Max", 0, 0, 255)
gen.add("v_max", int_t, 0, "Value Max", 0, 0, 255)

exit(gen.generate(PACKAGE, "FrontCamVision", "hsv"))
